data/json/recipes/armor/storage.json

```
@@ -387,6 +387,18 @@
     "qualities": [ { "id": "HAMMER", "level": 3 } ],
     "components": [ [ [ "rag", 6 ], [ "leather", 6 ], [ "cordage_short", 3, "LIST" ] ], [ [ "jerrycan", 1 ] ] ]
   },
+  {
+    "result": "boxpack",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "6 m",
```

```suggestion
    "time": "60 m",
```
There's no way you're making a box from scratch in five minutes.